### PR TITLE
Olensmar/docs/docusaurus and titles update

### PR DESCRIPTION
**Pull request by olensmar**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/articles/cicd-overview.mdx [hunk 1]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
 
 # Using Testkube from CI/CD
 
-You can trigger tests in Testkube from your CI/CD pipeline. This practice decouples running your tests from the CI/CD, removes all the complexity of, and speeds up your CI.
+Triggering Workflows from your CI/CD pipeline can be done in several ways:
+- Using one of Testkubes native integrations (see below)
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:6]:
```suggestion
- Using one of Testkubes native integrations (see below).
```

File: docs/articles/cicd-overview.mdx [hunk 2]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+- Automating the [Testkube CLI](/articles/install/cli) in your CI/CD tool
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:7]:
```suggestion
- Automating the [Testkube CLI](/articles/install/cli) in your CI/CD tool.
```

File: docs/articles/cicd-overview.mdx [hunk 3]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+- Directly calling the [Testkube API](/openapi/overview)
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:8]:
```suggestion
- Directly calling the [Testkube API](/openapi/overview).
```

File: docs/articles/cicd-overview.mdx [hunk 4]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+## Native CI/CD Integrations
+
+Native Integrations are available for
+- Jenkins - https://plugins.jenkins.io/testkube-cli
+- GitHub - https://github.com/marketplace/actions/testkube-action
+- Azure Devops - https://marketplace.visualstudio.com/items?itemName=Testkube.testkubecli
+
+:::tip
+
+The Workflow Details tab in the Testkube Dashboard shows you preconfigured configurations for popular CI/CD tools
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:32]:
```suggestion
The Workflow Details tab in the Testkube Dashboard shows preconfigured configurations for popular CI/CD tools
```

File: docs/articles/cicd-overview.mdx [hunk 5]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+to run the selected Workflow - [Read More](/articles/testkube-dashboard-workflow-details#cicd-integrations-tab)
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:33]:
```suggestion
to run the selected Workflow - [Read More](/articles/testkube-dashboard-workflow-details#cicd-integrations-tab).
```

File: docs/articles/cicd-overview.mdx [hunk 6]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+
+:::
+
+## General Steps for CI/CD Integration
+
+### 1. Utilize the Testkube CLI Docker Image
+
+In your CI/CD pipeline configuration, specify the use of the `kubeshop/testkube-cli` Docker image.
+
+This image contains the necessary Testkube CLI for interacting with Testkube services - [Read More](/articles/testkube-cli-docker)
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:43]:
```suggestion
This image contains the necessary Testkube CLI for interacting with Testkube services - [Read More](/articles/testkube-cli-docker).
```

File: docs/articles/cicd-overview.mdx [hunk 7]
```diff
@@ -2,16 +2,78 @@ import LegacyWarning from "../articles/_legacy-warning.mdx";
...
+Finally, add a script step to trigger the desired Workflow in Testkube. For example:
+
+```
+testkube run testworkflow <workflow-name> -f
+```
+
+Replace `<workflow-name>` with the specific name of the Test Workflow you wish to execute.
+
+:::tip
+You can run any Testkube CLI command using the approach described above, check out the [CLI Command Reference](/cli/testkube)
```
Comment (jfermi) [inline docs/articles/cicd-overview.mdx:78]:
```suggestion
You can run any Testkube CLI command using the approach described above, check out the [CLI Command Reference](/cli/testkube).
```

File: docs/articles/logs-and-artifacts.md [hunk 8]
```diff
@@ -26,6 +26,11 @@ For example, below shows the HTML report available in the screenshot above:
 
 ![Rendered Workflow HTLM Artifact](../img/workflows-htlm-artifact.png)
 
+:::tip
+
+You can retrieve logs and artifacts using the Testkube CLI also using the [get testworkflowexecution](/cli/testkube_get_testworkflowexecution)
```
Comment (jfermi) [inline docs/articles/logs-and-artifacts.md:31]:
```suggestion
You can retrieve logs and artifacts using the Testkube CLI also using the [get testworkflowexecution](/cli/testkube_get_testworkflowexecution).
```

File: docs/articles/scheduling-tests.mdx [hunk 9]
```diff
@@ -1,207 +1,57 @@
...
-<TabItem value="dashboard" label="Dashboard" default>
+## Scheduling a Workflow by updating the Workflow Definition
 
-## Create a Scheduled Test or Test Suite from the Testkube Dashboard
+:::info
 
-In order to run Tests and Test Suites on a regular basis, we support a scheduling mechanism for these objects.
-CRDs both for Tests and Test Suites contain a **schedule** field used to define rules for launching them in time.
-Testkube's schedule data format is the same that is used to define Kubernetes Cron jobs (check [Wikipedia Cron format](https://en.wikipedia.org/wiki/Cron) for details.
+Schedules are defined in the Workflow YAML under a `spec.events.cronjob` property - [Read More](/articles/test-workflows#cronjob)
```
Comment (jfermi) [inline docs/articles/scheduling-tests.mdx:9]:
```suggestion
Schedules are defined in the Workflow YAML under a `spec.events.cronjob` property - [Read More](/articles/test-workflows#cronjob).
```

File: docs/articles/scheduling-tests.mdx [hunk 10]
```diff
@@ -1,207 +1,57 @@
...
-Last Schedule Time:  Wed, 13 Apr 2022 15:50:00 +0300
-Active Jobs:         scheduled-test-tests-27497570
-Events:
-  Type    Reason            Age                  From                Message
-  ----    ------            ----                 ----                -------
-  Normal  SuccessfulCreate  5m41s                cronjob-controller  Created job scheduled-test-tests-2749757
-```
+:::tip
+Testkube's schedule data format is the same that is used to define Kubernetes Cron jobs,
+check [Wikipedia Cron format](https://en.wikipedia.org/wiki/Cron) for details.
```
Comment (jfermi) [inline docs/articles/scheduling-tests.mdx:31]:
```suggestion
visit [Wikipedia Cron format](https://en.wikipedia.org/wiki/Cron) for details.
```

File: docs/articles/scheduling-tests.mdx [hunk 11]
```diff
@@ -1,207 +1,57 @@
...
-```
-
-The test is successfully executed on the stated schedule.
+![Workflow Scheduling Definition](images/workflow-scheduling-definition.png)
 
-## Create a Test Suite with a Schedule
+:::info
 
-Let's create a Test Suite with a required schedule using the Testkube CLI command:
+The Dashboard will only allow you to manage _one_ schedule, if you want to define multiple schedules you will have
```
Comment (jfermi) [inline docs/articles/scheduling-tests.mdx:46]:
```suggestion
The Dashboard will only allow you to manage _one_ schedule, if you want to define multiple schedules, you will have
```

File: docs/articles/test-workflows.md [hunk 12]
```diff
@@ -147,6 +147,12 @@ spec:
       annotations:
         key4: value4
 ```
+
+:::info
+Testkube's schedule data format is the same that is used to define Kubernetes Cron jobs, 
+check [Wikipedia Cron format](https://en.wikipedia.org/wiki/Cron) for details.
```
Comment (jfermi) [inline docs/articles/test-workflows.md:153]:
```suggestion
visit [Wikipedia Cron format](https://en.wikipedia.org/wiki/Cron) for details.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Comment by olensmar** [comment]
thanks @jfermi !

**Pull request merged**